### ISSUE TITLE
start script picks up an IP address which is commented out

### ISSUE THREAD

**Issue by kuenishi** [basho/node_package#105]
In case when IP address is commented out in `riak.conf` like this:

```
## Name of the riak node
#nodename = riak@127.0.0.1
nodename = riak@192.168.62.159
```

then the `bin/riak` command grabs `127.0.0.1` as its IP address and the `start` and `ping` fails with timeout.

So the fix would be in [env.sh](https://github.com/basho/node_package/blob/develop/priv/base/env.sh#L59) like this:

``` sh
- NODENAME=`egrep 'nodename[ \t]*=[ \t]*' $RUNNER_ETC_DIR/riak.conf 2> /dev/null | cut -d = -f 2`
+ NODENAME=`egrep '^nodename[ \t]*=[ \t]*' $RUNNER_ETC_DIR/riak.conf 2> /dev/null | cut -d = -f 2`
```

**Comment by joedevivo** [comment]
Fixed in PR #106

**Comment by kuenishi** [comment]
Great! Thank you.
